Gopkg.toml

```diff
@@ -30,17 +30,21 @@
   branch = "master"
   name = "github.com/giantswarm/microerror"
 
+[[constraint]]
+  branch = "master"
+  name = "github.com/fortytw2/leaktest"
+
 [[constraint]]
   name = "k8s.io/client-go"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
 [[constraint]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.13.2"
+  version = "kubernetes-1.15.4"
 
-[[constraint]]
-  branch = "master"
-  name = "github.com/fortytw2/leaktest"
+[[override]]
```

Comment:
Do we remove this override later? Or why is it needed actually?

Reply:
It's a transitive dependency, but for unknown reason, updating direct dependencies (`k8s.io/client-go` and  `k8s.io/apimachinery`) was not enough, `dep ensure -update` simply was not updating previously vendored `k8s.io/api` version (and builds would fail), and then this little override helped to convince dep to do the right thing. I consider it to be harmless, since it's not adding that dependency as direct dependency. Could add a note, that it's there just to force transitive dependency version and can be removed once dep is fixed or we switch to Go modules once it actually is usable (currently not the case, has issues with k8s libraries versioning not following semver).

Reply:
I think this is because those branches don't have Gopkg.toml/lock files. Since the package isn't used in the project than dep just ignores it unless it's `override`.